Update android env vars (#849)
mattermost/mattermost-developer-documentation: site/content/contribute/mobile/build-your-own/android.md

| Variable            | Description                                | Default    | Required      |
|---------------------|--------------------------------------------|------------|---------------|

> | `COMMIT_CHANGES_TO_GIT` | Should the Fastlane script ensure that there are no changes to Git before building the app and that every change made during the build is committed back to Git. <br><br>Valid values are: `true`, `false` | `false` | No |

Reviewer: The COMMIT_CHANGES_TO_GIT description reads awkwardly and isn't clear -- starting the sentence with "Should the..." suggests a question is coming instead. Consider updating for clarity.

Author: @cwarnermm I'll leave that to someone who can provide clarification, this PR is only about removing the extra `\` on the variable names.

Author: BW #850 is also about the same thing

Reviewer: Fair. Approved intended changes via https://github.com/mattermost/mattermost-developer-documentation/pull/850 and noted that a couple of definitions & consistency issues remain.

Reviewer 2: I've reviewed [#850](https://github.com/mattermost/mattermost-developer-documentation/pull/850) and provided some comments. Perhaps we can sync both PRs up to include my feedback and Carrie's - that way they'll be aligned in terms of tense and so on. Happy to do that if it works for everyone.

Reviewer: That would be great, @justinegeffen -- Thank you so much! May I transfer https://mattermost.atlassian.net/browse/MM-36397 to you for completion?

| `BRANCH_TO_BUILD` | Defines the Git branch that is going to be used for generating the build. <br><br>**Make sure that, if this value is set, the branch it is set to exists**.| `$GIT_BRANCH` | No |
| `GIT_LOCAL_BRANCH` | Defines the local branch to be created from `BRANCH_TO_BUILD` to ensure the base branch does not get any new commits on it. <br><br>**Make sure a branch with this name does not yet exist in your local Git repository**. | build | No |
| `RESET_GIT_BRANCH` | Defines if, once the build is done, the branch should be reset to the initial state before building and whether to also delete the branch created to build the app. <br><br>Valid values are: `true`, `false` | `false` | No |

> | `VERSION_NUMBER `| Set the version of the app at build time to a specific value, rather than using the one set in the project. |  | No |

Reviewer: ```suggestion
| `VERSION_NUMBER `| Sets the version of the app at build time to a specific value, rather than using the one set in the project. |  | No |
```

> | `INCREMENT_VERSION_<br>NUMBER_MESSAGE` | Set the commit message when changing the app version number. | Bump app version number to | No |

Reviewer: ```suggestion
| `INCREMENT_VERSION_<br>NUMBER_MESSAGE` | Sets the commit message when changing the app version number. | Bump app version number to | No |
```

| `INCREMENT_BUILD_NUMBER` | Defines if the app build number should be incremented. <br><br>Valid values are: `true`, `false` | `false` | No |

> | `BUILD_NUMBER` | Set the build number of the app at build time to a specific value, rather than incrementing the last build number. |  | No |

Reviewer: ```suggestion
| `BUILD_NUMBER` | Sets the build number of the app at build time to a specific value, rather than incrementing the last build number. |  | No |
```

| `INCREMENT_BUILD_<br>NUMBER_MESSAGE` | Set the commit message when changing the app build number. | Bump app build number to | No |
| `APP_NAME` | The name of the app as it is going to be shown on the device home screen. | Mattermost Beta | Yes |
| `APP_SCHEME` | The URL naming scheme for the app as used in direct deep links to app content from outside the app. | mattermost | No |

> | `REPLACE_ASSETS` | Override the assets as described in [White Labeling](contribute/mobile/build-your-own/white-label/). <br><br>Valid values are: `true`, `false` | `false` | No |

Reviewer: ```suggestion
| `REPLACE_ASSETS` | Overrides the assets as described in [White Labeling](contribute/mobile/build-your-own/white-label/). <br><br>Valid values are: `true`, `false` | `false` | No |
```

| `MAIN_APP_IDENTIFIER` | The package identifier for the app. | | Yes |
| `BUILD_FOR_RELEASE` | Defines if the app should be built in release mode. <br><br>Valid values are: `true`, `false` <br><br>**Make sure you set this value to true if you plan to submit this app Google Play or distribute it in any other way**. | `false` | Yes |

> | `SEPARATE_APKS` | Build one APK per achitecture (armeabi-v7a, x86, arm64-v8a and x86_64) as well as a universal APK. The advantage is the size of the APK is reduced by about 4MB. <br><br>People will download the correct APK from the Play Store based on the CPU architecture of their device. | `false` | Yes |

Reviewer: ```suggestion
| `SEPARATE_APKS` | Builds one APK per achitecture (armeabi-v7a, x86, arm64-v8a and x86_64) as well as a universal APK. The advantage is the size of the APK is reduced by about 4MB. <br><br>People will download the correct APK from the Play Store based on the CPU architecture of their device. | `false` | Yes |
```

> | `SUBMIT_ANDROID_TO_<br>GOOGLE_PLAY` | Should the app be submitted to the Play Store once it finishes building, use along with `SUPPLY_TRACK`.<br><br>Valid values are: `true`, `false` | `false` | Yes |

Reviewer: Here is another instance of a description starting with "Should the..." that needs revision for clarity.

> | `SUPPLY_TRACK` | The track of the application to use when submitting the app to Google Play Store. Valid values are: `alpha`, `beta`, `production` <br><br>**RIt is not recommended to submit the app to production. First try any of the other tracks and then promote your app using the Google Play console**. | `alpha` | Yes |

Reviewer 2: ```suggestion
| `SUPPLY_TRACK` | The track of the application to use when submitting the app to Google Play Store. Valid values are: `alpha`, `beta`, `production` <br><br>**It's not recommended to submit the app to production. First try any of the other tracks and then promote your app using the Google Play console**. | `alpha` | Yes |
```

> | `SUPPLY_PACKAGE_NAME` | The package Id of your application, make sure it matches `MAIN_APP_IDENTIFIER`. |  | Yes |

Reviewer 2: ```suggestion
| `SUPPLY_PACKAGE_NAME` | The package ID of your application, make sure it matches `MAIN_APP_IDENTIFIER`. |  | Yes |
```
